wallet-new/src/Cardano/Wallet/Orphans/Arbitrary.hs

```diff
@@ -25,3 +28,30 @@ instance Arbitrary CWalletRedeem
 instance Arbitrary CPaperVendWalletRedeem
 instance Arbitrary CInitialized
 instance Arbitrary NewBatchPayment
+
+instance Arbitrary WalletStateSnapshot where
+    arbitrary = WalletStateSnapshot <$> arbitrary
```

Comment:
My gut feeling is that if we do not use `Swagger` in `src`, we don't need this `Arbitrary` instance at all.

Reply:
Where would you define instance of `Arbitrary WalletStateSnapshot` instead?

Reply:
That was a red herring, didn't realise `Example` has an explicit constrain on `Arbitrary` (for good reasons). My bad!